src/app/fdctl/run/tiles/fd_quic.c

```diff
@@ -619,8 +618,7 @@ privileged_init( fd_topo_t *      topo,
 
   FD_SCRATCH_ALLOC_INIT( l, scratch );
```

Comment:
Don't need FD_SCRATCH_...

Reply:
`l` is used later

Reply:
Was referring to `ctx` being unused, you can ignore the return code. But this part is wrong anyway and will cause memory corruption, the order of the FD_SCRATCH_* must have the same order for any caller that derives them from `void * scratch` so

* Remove line 132 above `l = FD_LAYOUT_APPEND( l, fd_ip_align(), fd_ip_footprint( 256UL, 256UL ) );`
* Remove this line 620
* Rebase and the fd_alloc is removed as well, so you don't need ctx at all in this function.
* Remove like 654 below `FD_SCRATCH_ALLOC_APPEND( l, fd_ip_align(), fd_ip_footprint( 256UL, 256UL ) );`